diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -138,26 +140,41 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
         } else {
             elseFromExistingCode
         }
-        val numberOfStatementsInElseBlock = if (isEqualToNull) {
-            (condition.treeParent.psi as KtIfExpression).then?.blockExpressionsOrSingle()?.count() ?: 0
-        } else {
-            (condition.treeParent.psi as KtIfExpression).`else`?.blockExpressionsOrSingle()?.count() ?: 0
-        }
 
-        val elseEditedCodeLines = getEditedElseCodeLines(elseCodeLines, numberOfStatementsInElseBlock)
+        val (numberOfStatementsInElseBlock, isAssignmentInNewElseBlock) = (condition.treeParent.psi as KtIfExpression)
+            .let {
+                if (isEqualToNull) {
+                    it.then
+                } else {
+                    it.`else`
+                }
+            }
+            ?.blockExpressionsOrSingle()
+            ?.let { elements ->
+                elements.count() to elements.any { element ->
+                    KtPsiUtil.isAssignment(element)
+                }
+            }
+            ?: Pair(0, false)
```

Comment:
I believe `0 to false` is more idiomatic.

Reply:
Well, it is either `(0 to false)` or `Pair(0, false)`. In this case second variant seem to be more clear